doc/source/metadata/Reference.rst

```diff
@@ -0,0 +1,577 @@
+and has the ``standard_name`` attribute set to ``forecast_period``.
+
+Ideally, this **should no longer** be used for IMPROVER data.
+
+.. warning::
+
+    Use of ``forecast_period`` in IMPROVER is deprecated
+    as it is at best unhelpful and at worst it is confusing,
+    as IMPROVER generates a blend from multiple sources
+    with different start times so there is no unique data time.
+
+grid_mapping
+************
+
+This CF attribute provides a label pointing to a separate 
+grid mapping variable, which more fully describes the map projection. 
+
+This **must** be present for gridded data,
+as **must** the associated grid mapping variable.
+
+height
+******
+
+This CF scalar coordinate variable is included in some
+cases to fully describe the quantity of interest.
+
+This **should** be included if there is any ambiguity in the interpretation
+of quantity of interest if it is omitted.
+(e.g. an inclusion of ``height`` with a value of ``1.5 m``
+for the representatiion of screen level.) 
+
+history
+*******
+
+Ideally, this netCDF attribute should provide a list of the applications
+that have modified the original data (i.e. an audit trail),
+with recommended practice being to add a date/time stamp
+(in the form ``YYYY-MM-DDThh:mm:ssZ``) and identify the software package.
+However, in practice, this is far from straightforward for IMPROVER
+as it processes a range of model runs,
+so there is no single, sequential processing chain
+from which to generate such an audit trail,
+making it impossible to accurately maintain previous history information.
+
+This is **not currently set** in IMPROVER.
+
+institution
+***********
+
+This CF attribute specifies where the original data was produced.
+
+This **must** be present and **should** take the name of the institution from
+where the data originated if only data from a single model has been processed.
+However, it **should** be set to the institution running the post-processing
+for multi-model blended data.
+
+latitude
+********
+
+This coordinate variable represents one half of the positional
+information for gridded data held on a
+Latitude-Longitude (strictly, equirectangular) projection.
+This is also used for site positions, which are only provided
+in latitude and longitude.
+It has the ``standard_name`` attribute set to ``latitude``
+and ``units`` set to ``degrees``.
+The latitude and longitude can be considered as relative the WGS84
+or the World Geodetic System 1984 datum,
+although this is not explicit in the metadata.
+
+This **must** be provided for site data and for gridded data
+on a Latitude-Longitude projection.
+For gridded data, if any statistical processing over the coordinate 
+has been applied,
+there **must** also be an associated ``latitude_bnds`` variable
+providing the bounds over which ``cell_methods`` are applied,
+although this is often included anyway to define the cell boundaries.
+The ``latitude_bnds`` variable has no attributes as it is tied to the 
```

Comment:
I think the sentence only actually references gridded data anyway - I've pulled it into a separate paragraph to make it clearer.

I say "if any statistical processing over the coordinate has been applied", so the cell_methods reference is just by way of expansion rather than being prescriptive.

I can't think of an example of spot data have spatial cell_methods.